### ISSUE TITLE
FR[cloud]: Set alias/open port from config file

### ISSUE THREAD

**Issue by barnabasbusa** [kurtosis-tech/kurtosis#2429]
### Background & motivation

It would be great if the aliases and open ports could be directly set from arguments file. 

This issue depends on #2428.

### Desired behaviour

Similar to issue #2428 it would be great to pass additional arguments in the yaml definition, such as aliases to be set automatically, and ports to be open automatically.

For example
```yaml
kurtosis:
  url: github.com/kurtosis-tech/ethereum-package
    branch: bbusa/test1
  verbosity: detailed
  image-download: always
  enclave: testing
  args-file: .
  alias: auto
  open-port:
    - dora-http
participants:
  - el_type: nethermind
    cl_type: lighthouse
network_params:  
  preset: minimal
additional_services: 
  - dora
  - tx_spammer

tx_spammer_params:
  tx_spammer_extra_args: ["--accounts=1", "--txcount=1"]
```

If an alias is set to auto, it could use the following logic:

Create an alias for service that has open-port definition, together with an enclave name:
for example: 
`dora-testing.cloud.kurtosis.com`

### How important is this to you?

Nice to have; this feature would make using Kurtosis more enjoyable.

### What area of the product does this pertain to?

Frontend: the Graphical User Interface, including the Enclave Manager
